modules/openapi-generator/src/main/resources/python/api_test.mustache

```diff
@@ -4,22 +4,20 @@
 
 import unittest
 
-import {{packageName}}
 from {{apiPackage}}.{{classFilename}} import {{classname}}  # noqa: E501
-from {{packageName}}.rest import ApiException
 
 
 class {{#operations}}Test{{classname}}(unittest.TestCase):
     """{{classname}} unit test stubs"""
 
-    def setUp(self):
-        self.api = {{apiPackage}}.{{classFilename}}.{{classname}}()  # noqa: E501
+    def setUp(self) -> None:
+        self.api = {{classname}}()  # noqa: E501
```

Comment:
The import block was importing *both* the package and the class; we only need one of these. It's more common in Python to use `from foo import Bar` style so that `Bar()` can be called without qualification.